test/fixtures/e2e/bento/bento-youtube.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Bento Youtube</title>
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <script
+      async
+      src="https://cdn.ampproject.org/v0/bento-youtube-1.0.js"
+    ></script>
```

Comment:
Update your branch, you now need `bento.js`.

```suggestion
    <script async src="https://cdn.ampproject.org/bento.js"></script>
    <script
      async
      src="https://cdn.ampproject.org/v0/bento-youtube-1.0.js"
    ></script>
```